python/semantic_kernel/connectors/openapi/sk_openapi.py

```diff
@@ -125,6 +121,9 @@ def prepare_request(
             content = self.request_body["content"]
             content_type = list(content.keys())[0]
             processed_headers["Content-Type"] = content_type
+            processed_headers["User-Agent"] = " ".join(
```

Comment:
the user agent header should be sent regardless whether the request has a body or not